lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -251,41 +257,56 @@ window.getDefaultAggregations = function() {
 
 /**
  * Save currently selected set of aggregation categories to local storage.
- * @param {!Array<{name: string, audits: !Array<string>}>} selectedAggregations
+ * @param {!{selectedAggregations: Array<{name: string, audits: !Array<string>}>,
```

Comment:
somewhere along the line this became wrong. I believe its `@param {{selectedAggregations: !Array<string>, disableExtensions: boolean}} settings`